apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -107,7 +108,7 @@ export class Utilities {
     const unresolvedUserFolder: string | undefined = process.env[
       (process.platform === 'win32') ? 'USERPROFILE' : 'HOME'
     ];
-    const homeFolder: string = path.resolve(unresolvedUserFolder);
+    const homeFolder: string = path.resolve(unresolvedUserFolder as string);
     if (!FileSystem.exists(homeFolder)) {
```

Comment:
We should probably have a defensive check here.

You can also say `unresolvedUserFolder!` to collapse the null check.

Reply:
Added defensive check.